PerformanceStatus.fsh

```diff
@@ -16,19 +16,21 @@ Description:    "The Karnofsky Performance Status (KPS) is a tool used to measur
 * basedOn only Reference (ServiceRequest | CarePlan)
 * partOf only Reference (Procedure)
 * category from http://hl7.org/fhir/ValueSet/observation-category (extensible)
-* code = LNC#89245-5 "Karnofsky Performance Status [Interpretation]"
+* code = LNC#89243-0 "Karnofsky Performance Status score"
 * subject only Reference(USCorePatient)
 * effective[x] only dateTime or Period
 * performer only Reference(Practitioner)
-* value[x] only CodeableConcept
-* valueCodeableConcept from KarnofskyPerformanceStatusVS (required)
+* value[x] only integer
```

Comment:
Since this is non-negative, change to `value[x] only unsignedInt` -- not a big deal, but an interesting case to see if FSH will handle it.

Reply:
Also, we could set an upper bound using the caret syntax.

Reply:
we could, although that would only work for ECOG.  We would need to further constrain Karnofsky so that the integers are in multiples of 10.  Not sure how that could be represented.

Reply:
Multiples of 10 isn't possible. But upper bound of 100 is possible, too.